README.md

```diff
@@ -8,17 +8,56 @@ A terraform provider for the [threefold grid](https://threefold.io) to manage yo
 - [Go](https://golang.org/doc/install) >= 1.15
 - [Gettting started document](https://library.threefold.me/info/manual/#/manual3_iac/grid3_terraform/manual__grid3_terraform_home)
 
-- to use the built plugin in a terraform file, use the following provider config:
+- to use the `mainnet`'s version of the provider for `v1.7.0`, use the following configs:
```

Comment:
can we please only mention mainnet and testnet only here in the README and move qa and dev relevant info into a separate page, or even have the full details in a separate page and copy the part of main/test only here ?